src/dymaptic.GeoBlazor.Core/Components/Widgets/ExpandWidget.cs

```diff
@@ -25,13 +26,15 @@ public class ExpandWidget : Widget
 
     /// <summary>
     ///     Icon font used to style the Expand button when collapsed.
+    ///     Deprecated
     /// </summary>
     [Parameter]
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public string? CollapseIconClass { get; set; }
```

Comment:
Do we want to simply remove these since they are deprecated?

Reply:
No, @AndersenBell we should use the C# `[Obsolete]` attribute for deprecated properties.